ts/a11y/explorer/KeyExplorer.ts

```diff
@@ -25,7 +25,7 @@
 
 import {A11yDocument, Region} from './Region.js';
 import {Explorer, AbstractExplorer} from './Explorer.js';
-import {sreReady} from '../sre.js';
+import {sreReady, SRE} from '../sre.js';
```

Comment:
I was a little confused by this, as `SRE` is a type in `sre.js`, but is being used as an object below.  Typescript should complain about that.

It turns out that `sre-node.ts` is the culprit.  Because it doesn't include an `import` or `export` command, it is not compiled as a module (in strict mode), so the `const SRE = require(...)` that it includes ends up being a global definition, not a local one, and the `sre-node.d.ts` ends up including a `declare const SRE` command.  Since Typescript processes all the `.d.ts` files (whether the `.js` file is included or not), that makes `SRE` a global even when `sre-node.js` isn't used.  That is why the use of `SRE` as a global is OK below.

This really needs to be fixed.  That could be done by adding

    export {};

to the bottom of `sre-node.ts` to force it to be treated as a module.  Then you could change

    export type SRE = {

below to

    export declare const SRE: {

in `sre.ts`.  That will make this `import` command import that `SRE` object, not the type, and all will be well, without the need for `sre-node.ts` to inadvertently create global variables.